### ISSUE TITLE
Can the text editor tab bar be configured to have multiple rows?

### ISSUE THREAD

**Issue by spinjector** [dbeaver/dbeaver#6504]
I tend to work with many tabs open, and I would like to move between tabs a bit more easily.

Can the text editor tab bar be configured to have multiple rows? 

The 'Show List' feature is ordered strangely. Can the sort order be configured to have the same order as the tabs, as they go from left-to-right..? Same thing for the Ctrl+Tab hotkey.

Having the connection name prepended to every tab caption is a big waste of space. Can the connection name be hidden on tabs..?

Thanks.

**Comment by serge-rider** [comment]
No, multiple rows are not supported. But you can use <kbd>CTRL+E</kbd> or <kbd>CTRL+SHIFT+E</kbd> for faster navigation.

Not sure I understand what is Show List..

Yes, you can remove conenction name from the tab:
![image](https://user-images.githubusercontent.com/6398845/62953000-ff9e3280-bdec-11e9-9921-de6510c7c28c.png)

**Comment by spinjector** [comment]
Oh, that's it: Ctrl+E is the 'Show List' I was speaking of. That's what it was called in the documentation. It's the order of that list that I find confusing. I think It sorts by modified items, and then alphabetically..? I have trouble with that; it would be much easier if it was sorted in the oder the tabs appear from let-to-right.

As for the tab names...bingo! The 'script title pattern' is what I was looking for. But now that I was able to explicitly go to it, I found it doesn't show up if I type 'title' or 'pattern' in the Preferences search box.

Thanks for your help. Reducing the script titles is a good step.
